test-dotnet/action.yml

```diff
@@ -44,44 +44,36 @@ runs:
 
     - name: Restore and cache private nuget packages
       uses: ./github-actions-test-dotnet/restore-dotnet
+      env:
+        DOTNET_NOLOGO: true
       with:
         dotnetVersion: ${{inputs.dotnetVersion}}
         packageReadonlyPat: ${{inputs.packageReadonlyPat}}
 
-    - name: .NET Build
-      shell: bash
-      run: |
-        for f in $(find . -name "*.sln"); do echo "restoring solution $f" && \
-          dotnet build $f --configuration Debug --no-restore
-        done
-
     - name: Configure AWS Credentials
       uses: aws-actions/configure-aws-credentials@v4.0.1
       with:
         aws-access-key-id: ${{inputs.awsAccessKeyId}}
         aws-secret-access-key: ${{inputs.awsAccessKeySecret}}
         aws-region: ${{inputs.awsRegion}}
-    
-    # GitHubActionsTestLogger nuget package should be installed in the test project 
+
+    # GitHubActionsTestLogger nuget package should be installed in the test project
     # coverlet.msbuild nuget package should be installed to generate coverage report
     - name: .NET Test
       shell: bash
-      run: |
-        for f in $(find . -name "*.Tests.csproj"); do echo "testing project $f" && \
-          csprojName=${f##*/} && \
-          projectName="${csprojName/.csproj/}" && \
-          dotnet test $f --configuration Debug --no-build --logger GitHubActions --collect "XPlat Code Coverage" --results-directory "TestResults/$projectName"
-        done
-    
+      run: dotnet test
```

Comment:
This loop through the test projects was requested by @monsieurleberre when this was refactored that time. I’d suggest to keep it.

Reply:
Happy to wait with PR open until @monsieurleberre  is back and confirms we still need this loop.

Reply:
https://github.com/trakx/github-actions/pull/26

I found the loop added here and it was because I was facing an issue back in February 2023:

`TestResults/7713c7e0-b73e-460e-b574-98de5ed27944/coverage.cobertura.xml generated an empty result.`

Similar issues haven't happened since.